packages/adapters/package.json

```diff
@@ -13,9 +13,12 @@
         "build": "dist"
     },
     "exports": {
-        ".": "./dist/@cornerstonejs/adapters.es.js"
+        ".": {
+            "types": "./dist/@cornerstonejs/dts/index.d.ts",
+            "import": "./dist/@cornerstonejs/adapters.es.js"
```

Comment:
it is recommended to use `.mjs` instead of this

Reply:
Can you make the change and test it with OHIF?  It isn't perfect yet, but I know it works right now with OHIF - will see if it still builds, but I think I'd like to stop searching for perfect in favour of good enough.

Reply:
fair enough, let's test these changes against other viewers. Will let you know